src/api-service/__app__/onefuzzlib/secrets.py

```diff
@@ -80,3 +80,8 @@ def delete_secret(secret_url: str) -> None:
     (vault_url, secret_name) = parse_secret_url(secret_url)
     keyvault_client = get_keyvault_client(vault_url)
     keyvault_client.begin_delete_secret(secret_name)
+
+
+def delete_secret_data(data: SecretData) -> None:
```

Comment:
Maybe it would be even clearer to name this something like:
```suggestion
def delete_remote_secret_data(data: SecretData) -> None:
```